DavidGonzalezZumaya/Collections/src/RansomNote/RansomeNote.java

```diff
@@ -0,0 +1,48 @@
+package RansomNote;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RansomeNote {
+
+    public static void checkMagazine(List<String> magazine, List<String> note) {
+
+        Map<String, Integer> magazineMap = createMapOfMagazine(magazine);
+
+        if(validateMessage(magazineMap, note)){
+            System.out.print("Yes");
```

Comment:
pull this out as a static final variable, because this could be used somewhere other plase